### ISSUE TITLE
Error importing calendar from iCloud

### ISSUE THREAD

**Issue by clochix** [cozy/cozy-calendar#298]
Reported by shark09 [in the forum](https://forum.cozy.io/t/bug-impossible-dimporter-un-calendrier/828).

He gets an exception trying to import an ICS file : 
```
/usr/local/cozy/apps/calendar/node_modules/cozy-ical/lib/index.js:505
      startDate: startDate.toDate(),
TypeError: Cannot call method 'toDate' of undefined
at VTodo.module.exports.VTodo.VTodo.extract (/usr/local/cozy/apps/calendar/node_modules/cozy-ical/lib/index.js:505:28)
at /usr/local/cozy/apps/calendar/node_modules/cozy-ical/lib/index.js:974:23
```

**Comment by prysme01** [comment]
Hello, I'm also facing this bug. The iCal comes from owncloud, and I can't import it into CozyCloud
Deprecation warning: moment().zone is deprecated, use moment().utcOffset instead. https://github.com/moment/moment/issues/1779
Error
    at deprecate (/usr/local/cozy/apps/calendar/node_modules/cozy-ical/node_modules/moment-timezone/node_modules/moment/moment.js:738:42)
    at /usr/local/cozy/apps/calendar/node_modules/cozy-ical/node_modules/moment-timezone/node_modules/moment/moment.js:2554:37
    at hookCallback (/usr/local/cozy/apps/calendar/node_modules/cozy-ical/node_modules/moment-timezone/node_modules/moment/moment.js:8:85)
    at Object.<anonymous> (/usr/local/cozy/apps/calendar/node_modules/cozy-ical/node_modules/moment-timezone/node_modules/moment/moment.js:11:2)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)

/usr/local/cozy/apps/calendar/node_modules/cozy-ical/lib/index.js:505
      startDate: startDate.toDate(),
                           ^
TypeError: Cannot call method 'toDate' of undefined
    at VTodo.module.exports.VTodo.VTodo.extract (/usr/local/cozy/apps/calendar/node_modules/cozy-ical/lib/index.js:505:28)
    at /usr/local/cozy/apps/calendar/node_modules/cozy-ical/lib/index.js:974:23
    at LineStream.<anonymous> (/usr/local/cozy/apps/calendar/node_modules/cozy-ical/lib/index.js:1004:11)
    at LineStream.emit (events.js:95:17)
    at LineStream.<anonymous> (_stream_readable.js:765:14)
    at LineStream.emit (events.js:92:17)
    at emitReadable_ (_stream_readable.js:427:10)
    at emitReadable (_stream_readable.js:423:5)
    at readableAddChunk (_stream_readable.js:166:9)
    at LineStream.Readable.push (_stream_readable.js:128:10)

**Comment by shark09** [comment]
Bug seems to be fixed on version 1.0.16.

thanks.

**Comment by aenario** [comment]
@shark09 thanks for reporting ! And sorry we did not notify you earlier.
